src/routes/search.js

```diff
@@ -4,13 +4,21 @@ const handler = (request, reply) =>{
   let searchQuery = encodeURIComponent(request.query.search);
   query.searchFor(searchQuery, (err,res) => {
     if(err) console.log(err);
-    let data = {
-        title: 'FACN Hapi Members',
-        description: 'An app which shows people involved in FACN1, where a user can see everyone involved, and add new people',
-        members: res.rows
-      }
-    reply.view('search', data);
-  });
+    else if(res.rows.length === 0){
+      let data = {
+          title: 'FACN Hapi Members',
+          description: 'An app which shows people involved in FACN1, where a user can see everyone involved, and add new people',
+        }
+      reply.view('noresult', data);
+    }
+    else{
```

Comment:
oh, true, might be inconsistent having 
```
if (err) {
  console.log(err);
  return;
}
```
in some places
and an else if in other places